src/accordion.test.visuals.ts

```diff
@@ -1,98 +1,85 @@
 import { expect, test } from '@playwright/test';
+import type GlideCoreAccordion from './accordion.js';
 
-test.describe('Accordion', () => {
-  test.describe('Light Mode', () => {
-    test('open', async ({ page }, test) => {
-      await page.goto('/iframe.html?id=accordion--accordion');
-
-      const handle = await page.waitForSelector('glide-core-accordion');
-
-      await handle?.evaluate((node) => {
-        node.open = true;
-        return node.updateComplete;
-      });
-
+test.describe('accordion--accordion', () => {
+  test.describe('globals=theme:light', () => {
+    test('open=${false}', async ({ page }, test) => {
+      await page.goto(`?id=${test.titlePath.at(1)}&${test.titlePath.at(2)}`);
+      await page.locator('glide-core-accordion').waitFor();
       await expect(page).toHaveScreenshot(`${test.titlePath.join('.')}.png`);
     });
 
-    test('closed', async ({ page }, test) => {
-      await page.goto('/iframe.html?id=accordion--accordion');
-      await page.waitForSelector('glide-core-accordion');
+    test('open=${true}', async ({ page }, test) => {
```

Comment:
I've gone back and forth on this, as you know. After zooming out, however, and seeing all the tests, I felt like "closed" and "open` just didn't fit it considering every test is necessarily named after the attribute under tests and doesn't attempt to describe the test. 

Additionally, every test name now maps directly to a Storybook control. So writing tests for a component is just a matter of doing down a story's control table. Most test suites took less than 5 minutes to write for these and other reasons.